lib/salus/package_utils/gemfile_lock.rb

```diff
@@ -0,0 +1,21 @@
+module Salus
+  class GemfileLock
+    def initialize(path)
+      @path = path
+    end
+
+    def add_line_number(cve)
+      pattern = case cve[:type]
+                when 'UnpatchedGem'
+                  cve[:name] + ' (' + cve[:version] + ')'
+                when 'InsecureSource'
+                  cve[:source]
+                end
+      if !pattern.nil?
+        line_in_gemfile_lock = `grep -n " #{pattern}" #{@path}`
```

Comment:
`IO.popen(["grep", "-n", pattern, @path]) ` 
would offer some protection against any malicious content... that's a small risk here but I think if someone hijacked the vuln database and injected a malicious string as the "version" we could pick it up here.   Probably paranoid to worry about it but I'm becoming a fan of IO.popen over backticks. 

Thoughts?  If I'm being too paranoid ignore me!